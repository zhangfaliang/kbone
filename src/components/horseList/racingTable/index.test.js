import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import RacingTable from './index';

describe('racingTable', async () => {
  let data = { title: 222, list: [], dataList: [] };
  it('racingTable', () => {
    expect.assertions(1);
    const wrapper = render(
      <RacingTable data={data} keys={{ title: 'location', information: 'list' }} item={() => {}} />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
