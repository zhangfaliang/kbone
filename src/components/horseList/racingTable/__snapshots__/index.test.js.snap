// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`racingTable racingTable 1`] = `null`;
